rust/connlib/shared/src/messages/client.rs

```diff
@@ -31,6 +35,33 @@ pub struct ResourceDescriptionCidr {
     ///
     /// Used only for display.
     pub name: String,
+
+    pub address_description: String,
+    pub gateway_groups: Vec<GatewayGroup>,
+}
+
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct GatewayGroup {
+    pub name: String,
+    pub id: SiteId,
+}
```

Comment:
Is a Gateway group different from a Site?

Reply:
I think it's just another name for site cc @AndrewDryga

Reply:
Can we call it `Site` then and rename the field using serde? Perhaps with a todo that the API should be changed?